src/hooks/useCioAutocomplete.ts

```diff
@@ -117,7 +117,7 @@ const useCioAutocomplete = (options: UseCioAutocompleteOptions) => {
             onSubmit({ query });
           }
           try {
-            cioClient?.tracker.trackSearchSubmit(query, { original_query: query });
+            cioClient?.tracker.trackSearchSubmit(query, { originalQuery: query });
```

Comment:
Not related to this PR but had to be updated in this PR because I updated client js version and types have been updated there